FirebaseRemoteConfig/Sources/RCNConfigDBManager.m

```diff
@@ -31,6 +31,7 @@
 #define RCNTableNameInternalMetadata "internal_metadata"
 #define RCNTableNameExperiment "experiment"
 
+static BOOL gIsNewDatabase;
```

Comment:
Does this need to be made thread-safe?

Reply:
it is. It runs on the database serial queue.